File: back/DevC/src/main/java/com/back/devc/domain/interaction/notification/dto/NotificationResponse.java
package com.back.devc.domain.interaction.notification.dto;

import java.time.LocalDateTime;

public record NotificationResponse(
        Long notificationId,
        Long userId,
        Long actorUserId,
        String actorNickname,
        Long postId,
        Long commentId,
        String type,
        String message,
        boolean isRead,
        LocalDateTime createdAt
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/interaction/notification/dto/NotificationResponse.kt
package com.back.devc.domain.interaction.notification.dto

import java.time.LocalDateTime

data class NotificationResponse(
    val notificationId: Long,
    val userId: Long,
    val actorUserId: Long,
    val actorNickname: String,
    val postId: Long?,
    val commentId: Long?,
    val type: String,
    val message: String,
    val isRead: Boolean,
    val createdAt: LocalDateTime,
)